openassessment/assessment/test/test_peer.py

```diff
@@ -859,14 +859,12 @@ def test_get_submitted_assessments(self):
         )
         self.assertEqual(assessment["points_earned"], 6)
         self.assertEqual(assessment["points_possible"], 14)
-        submitted_assessments = peer_api.get_submitted_assessments(bob_sub["uuid"], scored_only=True)
-        self.assertEqual(0, len(submitted_assessments))
```

Comment:
Since the only purpose of this assertion seemed to be testing the functionality of the not-used-anywhere-else `scored_only` argument, I removed it.